crates/header-translator/src/method.rs

```diff
@@ -909,3 +931,19 @@ pub(crate) fn handle_reserved(name: &str) -> String {
         name.into()
     }
 }
+
+pub(crate) fn apply_type_override(ty: &mut Ty, or: &TypeOverride) {
+    if let Some(nullability) = &or.nullability {
+        let c_nullability = match nullability {
+            config::Nullability::Nullable => clang::Nullability::Nullable,
+            config::Nullability::NonNull => clang::Nullability::NonNull,
+        };
+
+        match ty {
+            Ty::Pointer { nullability, .. } => {
+                *nullability = c_nullability;
+            }
```

Comment:
`Ty::Sel` and `Ty::IncompleteArray` has null-ability too. Could I get you to handle that as well?

Also, it'd be nice to emit an `warning!` for "nullabilities are equal / redundantly set in `translation-config.toml`".

Reply:
Yep, addressed :+1: